### PR TITLE
Review Me!

### PR DESCRIPTION
**Pull request by chaneberg**
So, I added an HTML page that makes a GET request from a static json file. Read the README for more details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # statler
```
Comment (bencdelaney) [inline README.md:1]: It looks like there's a conflict in this file. Could you merge the new changes from dev into your branch and re-commit

File: index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```
Comment (bencdelaney) [inline index.html:1]: It might be good to drop these files into a subdirectory (sandbox?) so we can see what belongs with what. Your call, though.

**Comment by mnijmeh11** [comment]
There are template for what  we are going so we can make it look good with a little effort. I will post those to an issue so you can check it out

**Comment by mnijmeh11** [comment]
Ok...thanks again. I still have issue with closing and opening up stuff on github

**Pull request merged**